db/re/item_db.conf

```diff
@@ -76066,7 +76069,22 @@ item_db: (
 	Type: 0
 	Buy: 0
 	Weight: 10
-	Script: <" itemheal rand(70,110),0; ">
+	Script: <"
+		.@temp = rand(1,4);
```

Comment:
temp is used in this item in aegis special.sc, that's why i make it temp also :)